script/lib/patches.py

```diff
@@ -3,19 +3,23 @@
 import codecs
 import os
 
+PATCH_DIR_PREFIX = "Patch-Dir: "
+PATCH_FILENAME_PREFIX = "Patch-Filename: "
+PATCH_LINE_PREFIXES = (PATCH_DIR_PREFIX, PATCH_FILENAME_PREFIX)
 
 def read_patch(patch_dir, patch_filename):
   """Read a patch from |patch_dir/filename| and amend the commit message with
   metadata about the patch file it came from."""
   ret = []
-  added_filename_line = False
+  added_patch_location = False
   patch_path = os.path.join(patch_dir, patch_filename)
   with codecs.open(patch_path, encoding='utf-8') as f:
     for l in f.readlines():
       line_has_correct_start = l.startswith('diff -') or l.startswith('---')
-      if not added_filename_line and line_has_correct_start:
-        ret.append('Patch-Filename: {}\n'.format(patch_filename))
-        added_filename_line = True
+      if not added_patch_location and line_has_correct_start:
+        ret.append('{}{}\n'.format(PATCH_DIR_PREFIX, patch_dir))
```

Comment:
If we're adding this by default- we should update existing patches in this PR to conform to the new formatting

Reply:
This doesn't get stored in the `.patch` files; it's only in the commit messages.

Reply:
ahhh gotcha! ty for the explanation 🙇‍♀️